components/test_raftstore/src/cluster.rs

```diff
@@ -1483,6 +1490,19 @@ impl<T: Simulator> Cluster<T> {
     pub fn partition(&self, s1: Vec<u64>, s2: Vec<u64>) {
         self.add_send_filter(PartitionFilterFactory::new(s1, s2));
     }
+
+    // set operation allowed level when tikv disk full happens.
+    pub fn set_disk_full_opt(&mut self, level: DiskFullOpt) {
```

Comment:
I think it's wrong. `DiskFullOpt` should be carried in `Context`. It shouldn't be a global variable for `Cluster`.

Reply:
Maybe this logic should be removed.